src/main/res/drawable/ic_bluetooth_searching_animated_24dp.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<animated-vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:drawable="@drawable/ic_bluetooth_searching_24dp">
```

Comment:
@dennisguse this was the problematic line in the stop-button animation. But I just tested this in an API 21 Emulator and the bluetooth icon was animated and didn't crash. Hmmm.